src/config_rmw.c

```diff
@@ -93,11 +93,10 @@ purge_after = %d\n"), DEFAULT_PURGE_AFTER) < 0)
 
 /* TRANSLATORS:  Do not translate the last line in this section  */
 if (fprintf (stream, _("\n\
-# purge will not run unless `--force` is used at the command line. Uncomment\n\
-# the line below if you would like purge to check daily for files that\n\
-# that exceed the days specified in purge_after\n\
+# If you'd like an extra saftey precaution, comment out the line below.\n\
+# Once disabled, purge will not run unless -f is used.\n\
 #\n\
-#force_not_required\n")) < 0)
+force_not_required\n")) < 0)
```

Comment:
I'll change this. Traditionally defaults are hard-coded into the program, so there should be  *commented* option here in the default config, "force_required". and the user can be advised to *uncomment* it for extra safety. That'll require me to change a little of the code.